Firestore/core/test/unit/model/field_path_test.cc

```diff
@@ -26,6 +28,12 @@ namespace firebase {
 namespace firestore {
 namespace model {
 
+namespace {
+FieldPath Parse(const std::string& path) {
```

Comment:
Optional nit: I'd add blank lines after the opening brace of the namespace and before the closing brace to make it a little easier to see the function separately, but it's just a personal preference -- feel free to ignore.

Reply:
Done